src/server/services/services-worker.js

```diff
@@ -104,7 +104,7 @@ class ServicesWorker {
     async loadRPCsFromDatabase() {
         await ServerStorage.onConnected;
         const CommunityServices = Storage.createCollection('netsblox:services:community');
-        const serviceData = await CommunityServices.find({}).toArray();
```

Comment:
This change can probably be removed now that your fix for the same issue is in the code.